modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/rest/api/server/organization/management/v1/OrganizationManagementBaseTest.java

```diff
@@ -0,0 +1,179 @@
+import org.testng.ITestContext;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.DataProvider;
+import org.wso2.carbon.automation.engine.context.TestUserMode;
+import org.wso2.identity.integration.test.rest.api.server.common.B2BRESTAPIServerTestBase;
+import org.wso2.identity.integration.test.rest.api.server.organization.management.v1.model.OrganizationType;
+
+import java.io.IOException;
+import java.util.Set;
+
+import static org.hamcrest.core.IsNull.notNullValue;
+import static org.wso2.identity.integration.test.rest.api.server.organization.management.v1.Utils.assertNotBlank;
+import static org.wso2.identity.integration.test.rest.api.server.organization.management.v1.Utils.extractOrganizationIdFromLocationHeader;
+
+/**
+ * Base test class for Organization Management REST APIs.
+ */
+public class OrganizationManagementBaseTest extends B2BRESTAPIServerTestBase {
+
+    private static final String API_DEFINITION_NAME = "org.wso2.carbon.identity.organization.management.yaml";
+    static final String API_VERSION = "v1";
+    public final OrganizationType organizationType;
+    static final String SUPER_ORGANIZATION_NAME = "Super";
+    static final String ORGANIZATION_NAME = "name";
+    static final String ORGANIZATION_PARENT_ID = "parentId";
+    static final String SUPER_ORGANIZATION_ID = "10084a8d-113f-4211-a0d5-efe36b082211";
+    protected String subOrganizationId;
+    protected static final String ORGANIZATION_MANAGEMENT_API_BASE_PATH = "/organizations";
+    protected static String swaggerDefinition;
+
+    static {
+        String apiPackageName = "org.wso2.carbon.identity.api.server.organization.management.v1";
+        try {
+            swaggerDefinition = getAPISwaggerDefinition(apiPackageName, API_DEFINITION_NAME);
+        } catch (IOException e) {
+            Assert.fail(String.format("Unable to read the swagger definition %s from %s", API_DEFINITION_NAME,
+                    apiPackageName), e);
+        }
+    }
+
+    public OrganizationManagementBaseTest(TestUserMode userMode, OrganizationType organizationType) throws Exception {
+
+        this.organizationType = organizationType;
+        super.init(userMode);
+        this.context = isServer;
+        this.authenticatingUserName = context.getContextTenant().getTenantAdmin().getUserName();
+        this.authenticatingCredential = context.getContextTenant().getTenantAdmin().getPassword();
+    }
+
+    @BeforeClass(alwaysRun = true)
+    public void init(ITestContext context) throws Exception {
+
+        ISuite suite = context.getSuite();
+        String orgId = (String) suite.getAttribute("createdOrgId");
+        if (orgId == null) {
+            orgId = SUPER_ORGANIZATION_ID;
+        }
+        this.subOrganizationId = orgId;
+        if (this.organizationType.name().equals("SUPER_ORGANIZATION")) {
```

Comment:
use a constant for "SUPER_ORGANIZATION"
Also, change the check as in the following format
<constants>.equals(<variable>)

Reply:
Fixed by - https://github.com/wso2/product-is/pull/16566/commits/177717a2f7457f7a3e486d6359b440b160ab1fe5